src/models/types/model.customer.ts

```diff
@@ -19,4 +19,16 @@ export interface UpdateCustomerModel {
   email?: string;
   phone?: string;
   direction?: Direction;
+}
+
+export interface CustomerDTO {
+  id?: Number;
+  name?: string;
+  username: string;
+  email: string;
+  phone: string;
+  direction: string;
+  password: string;
+  status: Number;
+  date: number;
```

Comment:
All these fields must be optional, because when you perform a server request you will get different values in each request. Make all this staff optional, if not your code will not work when we make filter server request. 
If you are not sure what does it mean please ask me.

Reply:
@DanBaDo @Alcazar87 This is **REALLY** important. Please both of you check this staff.